server/src/main/kotlin/org/web3j/openapi/server/Properties.kt

```diff
@@ -17,7 +17,7 @@ object Properties {
     const val PRIVATE_KEY = "web3j.openApi.privateKey"
     const val WALLET_FILE = "web3j.openApi.wallet.file"
     const val WALLET_PASSWORD = "web3j.openApi.wallet.password"
-
+    const val CONTRACT_ADDRESSES = "web3j.openApi.contract.addresses"
 //    const val WEB3J = "web3j.openApi.web3j"
```

Comment:
need to sort this out before merge - @xaviarias do you know anything about this commented out section?

Reply:
I think we could remove it but ask Rachid to be sure.